### PR TITLE
Docs: Project and Sample

### PR DESCRIPTION
**Pull request by ChrisHuynh333**
## What does this PR do and why?
This PR updates the Project and Sample docs. This includes:
- Projects Intro
- View Project History
- Samples intro
- View Samples History
- Download Sample Files
- Sample Metadata

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
1. Start the docs in a terminal within your `irida-next` repo with:
```
cd docs-site
pnpm build
npm run serve
```
2. Read the documentation and see if it makes sense from a user's perspective.
3. Check that links navigate to the correct pages.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: docs-site/docs/user/organization/members/group-and-project-members.md [hunk 1]
```diff
@@ -85,9 +85,10 @@ Prerequisite:
```
Comment (JeffreyThiessen) [inline docs-site/docs/user/organization/members/group-and-project-members.md:85]: Above this can you fix the wording of `Edit member for a group or project` to `Edit members for a group or project`

File: docs-site/docs/user/organization/members/group-and-project-members.md [hunk 2]
```diff
@@ -137,10 +139,11 @@ Prerequisites:
 
 To remove a member from a project:
 
-1. From the left sidebar, select **Projects**, and find your project
-2. From the left sidebar, select **Members**
-3. On the right hand side of the row for the member you want to remove, click the `ellipsis` and select **Remove** from the dropdown.
-4. Confirm that you would like to remove the member from the project in the popup by clicking the **OK** button
+1. From the left sidebar, select **Projects**
+2. Select the project
```
Comment (JeffreyThiessen) [inline docs-site/docs/user/organization/members/group-and-project-members.md:143]: Below this there is the `Remove a member from a group`, which should also have the wording changed like this.

File: docs-site/docs/user/project/projects/manage-projects.md [hunk 3]
```diff
@@ -75,7 +75,15 @@ To update the URL of the project:
...
 
 ## View Project History
-<!-- TODO: Add steps to view project history -->
+
+Prerequisite:
+- You must have at least the Maintainer role for the project.
+
+To view a project's history:
+
+1. Select the project that you want to edit and then on the left sidebar, select **History**
```
Comment (JeffreyThiessen) [inline docs-site/docs/user/project/projects/manage-projects.md:84]: Please break this into 2 steps like is done on the other page in the PR

**Review by JeffreyThiessen — CHANGES_REQUESTED** [review]
Looks great! Just a couple nitpicks

File: docs-site/docs/user/organization/members/group-and-project-members.md (hunk 1 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/organization/members/group-and-project-members.md:85]: Fixed in 52b5939cbce0f838bffc5b200d96f444c0d5756d

File: docs-site/docs/user/organization/members/group-and-project-members.md (hunk 2 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/organization/members/group-and-project-members.md:143]: Fixed in 52b5939cbce0f838bffc5b200d96f444c0d5756d

File: docs-site/docs/user/project/projects/manage-projects.md (hunk 3 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/project/projects/manage-projects.md]: Fixed in 52b5939cbce0f838bffc5b200d96f444c0d5756d

**Review by JeffreyThiessen — APPROVED** [review]
Looks great!

File: docs-site/docs/user/project/samples/sample-metadata.md [hunk 4]
```diff
@@ -4,10 +4,160 @@ id: sample-metadata
...
+
+    | Key            | Value  |
+    | :------------- | :----- |
+    | MetadataField1 | value4 |
+    | MetadataField2 | value5 |
+    | MetadataField3 | value6 |
+
+When creating the spreadsheet, you are required to have a column that contains a sample identifier. The identifier is case-sensitive and can contain either the sample names or IDs.
+
+**An important note:** When importing a metadata spreadsheet, you will be asked if you'd like to **Ignore empty values**. If this is **selected**, any metadata fields without an associated value will not be added to the sample. However, if this **not selected**, any samples with the metadata key and empty value will be **deleted**.
```
Comment (ericenns) [inline docs-site/docs/user/project/samples/sample-metadata.md:88]: **nitpick (blocking):** Can you rephrase the following:

`If this is **selected**, any metadata fields without an associated value will not be added to the sample.`

to:

`If this is **selected**, any metadata fields without an associated value will be ignored and those metadata keys will not be removed from the sample if present.`

**Review by ericenns — CHANGES_REQUESTED** [review]
Just one minor comment, otherwise it looks great

File: docs-site/docs/user/project/samples/sample-metadata.md (hunk 4 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/project/samples/sample-metadata.md]: Replaced in d55b07c09b33c3914f3f21c5a0f2ee10aa7e2734

File: docs-site/docs/user/project/projects/manage-projects.md [hunk 5]
```diff
@@ -75,7 +75,16 @@ To update the URL of the project:
 3. Select **Change project URL**
 
 ## View Project History
-<!-- TODO: Add steps to view project history -->
+
+Prerequisite:
+- You must have at least the Maintainer role for the project.
```
Comment (deepsidhu85) [inline docs-site/docs/user/project/projects/manage-projects.md:80]: Can you also add in, that the user must be a direct or inherited member from the project's ancestors. Shared groups (namespace group links) and their members cannot see the history

File: docs-site/docs/user/project/samples/manage-samples.md [hunk 6]
```diff
@@ -95,11 +100,23 @@ Prerequisites:
...
+2. Select the project
+3. From the left sidebar, select **Samples**
+4. Select the sample
+5. Select **Remove**
 
 ## View Sample History
 
-<!-- TODO: Add steps for sample history -->
+Prerequisite:
+- You must have at least the Maintainer role for the sample's project.
```
Comment (deepsidhu85) [inline docs-site/docs/user/project/samples/manage-samples.md:112]: Same comment as above about only direct or inherited members can see the history

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
This looks great just a couple of small comments below

**Review by JeffreyThiessen — APPROVED** [review]
Looks great!

File: docs-site/docs/user/project/samples/manage-samples.md (hunk 6 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/project/samples/manage-samples.md:112]: Added in e224fa502c5bc1af0101a56f5be2e42f97937f52

File: docs-site/docs/user/project/projects/manage-projects.md (hunk 5 above)
Comment (ChrisHuynh333) [reply docs-site/docs/user/project/projects/manage-projects.md:80]: Added in 5d6e652c3e18dae83fd0b12b8b61e8b5df717799

**Review by deepsidhu85 — APPROVED** [review]
